src/main/java/com/manardenza/servise/HotelService.java

```diff
@@ -0,0 +1,46 @@
+package com.manardenza.servise;
+
+import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.Hotel;
+
+import java.util.List;
+import java.util.Map;
+
+public final class HotelService {
+
+    private static HotelService instance;
+    private HotelDaoImpl hotelDao;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "hotelDao" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)